vagrant/provisioning/roles/arkcase-app/templates/arkcase-commons-server.yaml

```diff
@@ -1,8 +1,8 @@
 arkcase:
   baseURL: "https://{{ external_host }}/arkcase"
   basePortalURL: "https://{{ external_host }}/portal"
-  fileManagementURL: "https://{{ internal_host }}:{{ file_management_port | default(9068) }}"
-  fileUtilsURL: "https://{{ internal_host }}:{{ file_utils_port | default(9078) }}"
+  fileManagementURL: "http://{{ internal_host }}:{{ file_management_port | default(9068) }}"
```

Comment:
Can you take the protocol from another fact, e.g. `microservice_protocol` or `file_management_protocol`?  That way when I enable TLS on FedRAMP, I only have to update the fact.